docs/Product-Guides/Assisted-Connect-Flow.md

```diff
@@ -1,45 +1,42 @@
+Contact the Finch team at `developers@tryfinch.com` to setup your application to work with Assisted Connect Flow.
 
-### Launch Connect with the `manual` flag
+### Launch Finch Connect with the `manual` flag
 <!-- theme: info -->
-> For Assisted Connect [providers](../Development-Guides/Providers.md) on Connect, your users will see instructions on how to manually connect their employment systems to Finch and your application.
-
-Launch [Connect](../Integrating-with-Finch/Integrate-Finch-Connect/Redirect-to-Connect.md) with `manual=true` so your user can view all the Assisted Connect providers Finch supports. After granting your application access, you can retrieve an authorization `code` and exchange it for an `access_token` as usual.
+> For [Assisted API Providers](../Development-Guides/Providers.md#assisted-api-providers) on Finch Connect, your users will see instructions on how to manually connect their employment systems to Finch and your application.
 
+Launch [Finch Connect](../Integrating-with-Finch/Integrate-Finch-Connect/Redirect-to-Connect.md) with `manual=true` so your user can view all the Assisted API providers Finch supports. After granting your application access, you can retrieve an authorization `code` and exchange it for an `access_token` as usual.
 
 ### Handle API responses correctly
 #### `202` status code response
-With automated providers, you are able to create an `access_token` and immediately use it to pull in data or push changes to your users' systems. However, with Assisted Connect providers, there is a short delta from when the `access_token` is created to when you can do anything with it.
+With [Automated API Providers](../Development-Guides/Providers.md#automated-api-providers), you are able to create an `access_token` and immediately use it to pull in data or push changes to your users' systems. However, with [Assisted API Providers](../Development-Guides/Providers.md#assisted-api-providers), there is a short delta from when the `access_token` is created to when you can do anything with it.
 
 Until the `access_token` is ready for use, Finch will return an empty response with the HTTP status code `202` indicating the request is valid but the processing is pending. When the `access_token` is ready, API calls will return data with the HTTP status code `200` as usual.
 
 #### `Finch-Data-Retrieved` header
 Since the data is retrieved and refreshed on a periodic cadence, you may want to know the freshness of the data returned by an API call. You can use the `Finch-Data-Retrieved` [header](../Development-Guides/Headers.md) to get the date and time in ISO-8601 format the data was retrieved.
 
-### Monitoring the status of your Assisted Connect connections 
-To view the status of your connected employers, you can use the [Finch Developer Dashboard](https://dashboard.tryfinch.com) and navigate to the **Employers** tab on your application's page. 
+### Monitoring the status of your Assisted API Connections 
```

Comment:
For all usages of Assisted API and Automated that are followed by a noun (e.g. connections, providers, benefit providers, etc.), I usually do not capitalize the following noun. E.g. I use `Assisted API connection` instead of `Assisted API Connection` since the product is `Assisted API`. But I honestly don't know which way is correct. Just wanted to share that perspective.

Reply:
I was wondering this. It wasn't consistent with our docs, so I just picked one way and went with it. It felt right in most instances and weird in others. 🤷‍♂️